docs/platforms/javascript/common/migration/v7-to-v8/v8-new-performance-api.mdx

```diff
@@ -0,0 +1,253 @@
+---
+title: New Performance Monitoring APIs
```

Comment:
Eventually this should live under `platforms/javascript/guides/node/performance/instrumentation/custom-instrumentation/`, but this is a good spot to throw all of this in until we finally release v8.